meta-resin-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -20,12 +22,12 @@ if [ "$local_image" =  "" ] && [ "$remote_image" = "" ]; then
 	exit 1
 fi
 
-export DOCKER_HOST="unix:///var/run/docker-host.sock"
+export DOCKER_HOST="unix:///var/run/balena-host.sock"
```

Comment:
So balena keeps the `DOCKER_HOSTS` env var to know how to talk to the daemon, that didn't change to `BALENA_HOST`?

Reply:
No, this was missed in the renaming